<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatClass</class>
 <widget class="QWidget" name="GroupChatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupChat</string>
  </property>
  <widget class="QLabel" name="groupChatLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  background-color: #4A4A8D; /* 背景色 */
    border-bottom: 1px solid #CCCCCC; /* 底部边框 */</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="chatListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>500</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="chatInputext">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
      background-color: red;
   }
QPushButton {
border-image: url(:/QQ/image/关闭 (1).png);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>435</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#sendButton {
    background-color: #4A4A8D;
    color: white;
    border-radius: 5px;
}
QPushButton#sendButton:hover {
    background-color: #005A9E;
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton:hover {
      background-color: red;
   }
QPushButton {
border-image: url(:/QQ/image/关闭 (1).png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="miniButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    QPushButton:hover {
      background-color: rgba(255, 255, 255, 50);
    }
QPushButton{
border-image: url(:/QQ/image/最小化_2.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="memberList">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>150</width>
     <height>430</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>500</width>
     <height>2</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>2</width>
     <height>430</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QQ.qrc"/>
 </resources>
 <connections/>
</ui>
